framework/ipc/src/org/apache/cloudstack/framework/serializer/OnwireClassRegistry.java

```diff
@@ -108,7 +107,7 @@ public void scan() {
                 while (resources.hasMoreElements()) {
                     String filePath = resources.nextElement().getFile();
                     // WINDOWS HACK
-                    if (filePath.indexOf("%20") > 0)
+                    if (filePath != null && filePath.indexOf("%20") > 0)
```

Comment:
There is already a filePath != null immediately below. The check can go inside that to avoid duplication.